tickets/src/config.js

```diff
@@ -27,6 +27,7 @@ export default function configure(
   const unlockTicketsUrl =
     runtimeConfig.unlockTicketsUrl || 'http://0.0.0.0:3003'
   let unlockAppUrl = 'http://0.0.0.0:3000'
+  let unlockStaticUrl = 'http://0.0.0.0:3000'
```

Comment:
I think the static site is actually on a different port. 
```suggestion
  let unlockStaticUrl = 'http://0.0.0.0:3002'
```